src/components/weight-table.tsx

```
@@ -0,0 +1,283 @@
+
+  // 列の値を取得するヘルパー関数
+  const getCellValue = (entry: DataEntry, columnId: string) => {
+    switch (columnId) {
+      case "weight":
+        const weight = getDisplayWeight(entry);
+        return weight ? formatValue(weight, '', 1) : '-';
+      case "fatRatio":
+        return entry.type === 'bodyComposition' && entry.fatRatio !== null 
+          ? formatValue(entry.fatRatio, '', 1) : '-';
+      case "muscleMass":
+        return entry.type === 'bodyComposition' && entry.muscleMass !== null 
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)